tests/cache/test_region.py

```diff
@@ -7,6 +7,8 @@
 from unittest import mock
 from unittest import TestCase
 
+from _pytest.python_api import approx
```

Comment:
mmmmm, cant use a private import like this..  (looks like [it has a public API](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx)), but see below where we dont need it

Reply:
[done](https://github.com/sqlalchemy/dogpile.cache/pull/247/commits/3bef2f827713fa4b2952be38d33c8a0caef3ae56) (removed)